invenio_communities/config.py

```diff
@@ -202,6 +202,22 @@
 """Community logo size quota, in bytes."""
 
 
+COMMUNITIES_NAMESPACES = {}
```

Comment:
this should be overwritten in rdm-records

`COMMUNITIES_NAMESPACES = RDM_NAMESPACES`